content/pt/_index.html

```diff
@@ -57,6 +57,64 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
 </div>
 {{< /blocks/section >}}
 
-{{< blocks/kubernetes-features >}}
+{{< blocks/kubernetes-features id="features" >}}
```

Comment:
Hi @jcjesus. Actually, this file shouldn't be changed. From what I can see this block `{{< blocks/kubernetes-features >}}` reads from the file https://github.com/kubernetes/website/blob/master/layouts/shortcodes/blocks/kubernetes-features.html

@remyleone do you have any input regarding the translation of this static blocks?